internal/controllers/topology/cluster/cluster_controller.go

```diff
@@ -30,6 +30,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
+	ctrlpredicates "sigs.k8s.io/controller-runtime/pkg/predicate"
```

Comment:
nit: I wonder if we should use `crpredicates` instead. I saw `cr` as prefix in other providers as prefix for controller-runtime and I think it's more intuitive and reads better.

Reply:
Whatever suits - I picked ctrl because it's used to import controller runtime itself in other parts of the code, and ctrl client is used in places. We should try to keep the usage consistent if we can.

Reply:
Yup agree. If nobody else comes up with an opinion on that, let's just leave it as is.
I think currently this would affect:
* ctrl => sigs.k8s.io/controller-runtime
* ctrlclient => "sigs.k8s.io/controller-runtime/pkg/client" (which is only very rarely aliased at all, but I don't see a problem there)
* ctrlpredicates "sigs.k8s.io/controller-runtime/pkg/predicate"